<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6,3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="DisablingListWidget" name="listFrames">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="pteNotes"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Note that convolution with IRF will smear out oscillations with frequencies higher that those corresponding to FWHM of excitation pulse (which is about 1300/cm for FWHM of 25fs) and even lower frequencies will have smaller amplitude.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FigureWidget" name="plotMain"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QComboBox" name="cbTRN">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Total</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rephasing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Non-Rephasing</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QSpinBox" name="iInterpolation1">
         <property name="toolTip">
          <string>Interpolation of ω₁ axis.</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Interpolation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="bGlobalColors">
         <property name="text">
          <string>Global</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QSpinBox" name="iInterpolation3">
         <property name="toolTip">
          <string>Interpolation of ω₃ axis.</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="cbPAIR">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Imag</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ampl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="bFixColors">
         <property name="text">
          <string>Fix</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1,1">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Population</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="GAInputList" name="listGAInput" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Positive frequency gives exp(-t2*(t+1j*w)), negative gives exp(-t2*(t-1j*w))&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,0,0,1,0,0,0">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TRN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbTRN_GA">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <item>
            <property name="text">
             <string>Total</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rephasing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Non-Rephasing</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushLimits">
           <property name="text">
            <string>Limits...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushUpdateGA">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushSwarm">
           <property name="text">
            <string>Swarm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushStopSwarm">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lMaxFreq">
           <property name="toolTip">
            <string>Note that a fitted +1800/cm component might be in fact -200/cm component if your max frequency is 2000/cm and you selected wrong sign for fitting.</string>
           </property>
           <property name="text">
            <string>Max frequency estimate</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lSSR">
           <property name="toolTip">
            <string>Note that a fitted +1800/cm component might be in fact -200/cm component if your max frequency is 2000/cm and you selected wrong sign for fitting.</string>
           </property>
           <property name="text">
            <string>Sum of residues**2</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushSSR">
           <property name="text">
            <string>Show SSR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushBootstrap">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Use utils/bootstrap instead!</string>
           </property>
           <property name="text">
            <string>Bootstrap</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>IRF </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>FWHM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dIRFFWHM">
         <property name="toolTip">
          <string>Set to zero to turn IRF off</string>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bIRFFWHMFix">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>offset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dIRFOffset">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Due to convolution the 0fs position is shifted from the peak maximum for the model (i.e. the model does not peak at 0fs). Therefore offset of about half pulse width is OK.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bIRFOffsetFix">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="FigureWidget" name="plotFitness"/>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabCuts">
        <attribute name="title">
         <string>Cuts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="FigureWidget" name="plotCuts"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="cbDisHorizontal">
             <property name="styleSheet">
              <string notr="true">color: #0000ff;</string>
             </property>
             <property name="text">
              <string>Horizontal</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="cbDisVertical">
             <property name="styleSheet">
              <string notr="true">color: #d50000;</string>
             </property>
             <property name="text">
              <string>Vertical</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="cbDisDiagonal">
             <property name="styleSheet">
              <string notr="true">color: #007c00;</string>
             </property>
             <property name="text">
              <string>Diagonal</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="cbDisDiagonalFit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Do not fit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lorentzian</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="labDisDiagonalFit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>μ: Position of peak relative to the tracer position
A: Peak maximum
FWHM: Full Width at Half Maximum</string>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="cbDisAntiDiagonal">
             <property name="styleSheet">
              <string notr="true">color: #7100aa;</string>
             </property>
             <property name="text">
              <string>Anti-Diagonal</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="cbDisAntiDiagonalFit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Do not fit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lorentzian</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="labDisAntiDiagonalFit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>μ: Position of peak relative to the tracer position
A: Peak maximum
FWHM: Full Width at Half Maximum</string>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabTimeline">
        <attribute name="title">
         <string>Timeline</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="FigureWidget" name="plotTimeline"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisablingListWidget</class>
   <extends>QFrame</extends>
   <header>common/widgets/DisablingListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FigureWidget</class>
   <extends>QGraphicsView</extends>
   <header>pqr/pqr2</header>
  </customwidget>
  <customwidget>
   <class>GAInputList</class>
   <extends>QWidget</extends>
   <header>common/widgets/GAInputList</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
